tsconfig.json

```diff
@@ -9,12 +9,11 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "sourceMap": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "typeRoots": ["./types", "./node_modules/@types"]
```

Comment:
:question: is this needed here? I guess this is needed for `mxgraph-type-definitions` introduction

Reply:
yes it is part of next PR, removing